cellmlmanip/model.py

```diff
@@ -451,10 +457,16 @@ def get_derivative_symbols(self):
         """
         return [v for v in self.graph if isinstance(v, sympy.Derivative)]
 
-    def get_state_symbols(self):
+    def get_state_symbols(self, order_by_order_added=False):
         """Returns a list of state variables found in the given model graph.
+        order_by_order_added indicates whether state_symbols are sorted in the order they appear in the model
+        (otherwise orderin is determined by the other in equations)
```

Comment:
sp ordering

Reply:
not sure what this sentence means ? or should 'other' be 'order'

Reply:
yes & fixed